sui_core/src/gateway_state.rs

```diff
@@ -242,6 +242,7 @@ where
             .ok_or(SuiError::ObjectNotFound {
                 object_id: *object_id,
             })?;
+        debug!(?object_id, ?object, "Fetched object");
```

Comment:
btw this only reads from local store, is that worth a log? If so, maybe make it more explicit as fetch object from local store.